dhcpcd-hooks/65-kano-dhcp-hook

```diff
@@ -38,24 +36,60 @@ REBOOT | BOUND)
     # Try uploading the tracking data to our servers
     # Should be quiet on failure
     kano-sync --upload-tracking-data --silent &
+}
 
-    # Sync objects from the content API in the background
-    # TODO: This should be removed once we have the daemon done
+# Sync objects from the content API in the background
+# TODO: This should be removed once we have the daemon done
+kano_daemon () {
     kano-content sync &
-
     kano-tracker-ctl +1 'internet-connection-established' &
-    ;;
+}
 
-NOCARRIER | STOPPED)
-    # Are we still networked due to multihomed connection?
+# Check for a multihomed connection and keep it if there
+# otherwise disconnect with a logger notification
+is_multihomed () {
     is_internet
     if [ $? -eq 0 ]; then
-    	logger "kano-dhcp-hook: NOCARRIER/STOPPED but still connected, aborting action"
+    	logger "kano-dhcp-hook: $reason but still connected, aborting action."
     	exit 0
     else
-        logger "kano-dhcp-hook: NOCARRIER/STOPPED and connectivity lost"
+        logger "kano-dhcp-hook: $reason and connectivity lost."
         rm $monitor_file
     fi
+}
+
+# In case of lease RENEWal or REBOUND to a new DHCP server
+# check if connection is sucesfully active and modifly
+# the monitor_file accordingly
+is_reconnected () {
+    if [$? -eq 0]; then
```

Comment:
Maybe the `is_internet` call should be moved into this function so that the `$?` always makes sense.